UnitsNet/BaseUnits.cs

```diff
@@ -101,6 +101,21 @@ public bool IsSubsetOf(BaseUnits other)
                 (LuminousIntensity == LuminousIntensityUnit.Undefined || LuminousIntensity == other.LuminousIntensity);
         }
 
+        /// <summary>
+        /// Checks if all of the base units are defined.
+        /// </summary>
+        /// <returns>True if all of the base units are defined, otherwise false.</returns>
+        public bool AreAllBaseUnitsDefined()
```

Comment:
Since BaseUnits is readonly, this could be a getter property initialized once or lazily. Prop name idea: `IsFullyDefined`

Reply:
👍 I like it better.

No need for lazy. Seven && checks is not expensive 😄 I'll just make it a property set in the constructor.